src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -171,6 +171,9 @@ image::execute.gif[100000,2000]
 The ActiveEon Jupyter Kernel adds a kernel backend to Jupyter. This kernel interfaces directly with the ProActive
 scheduler and constructs tasks and workflows to execute them on the fly.
 
+With this interface, a user can run his code locally and test it using a native python kernel, and by a simple switch to
+ProActive kernel, run it on remote public/private infrastructures without having to modify his code. See the example below:
+
```

Comment:
Using this interface, users can run their code locally and test it using a native python kernel. For instance, a simple switch to the ProActive kernel allows running code on remote public or private infrastructures without having to modify the code. See the example below:

Reply:
Sorry but I don't agree here neither. Your suggestion changes the meaning of the original sentence. I will refactor it.
Thanks!

Reply:
Your sentence is too long. Please consider rephrasing it into multiple ones.